docs/user/troubleshooting.md

```diff
@@ -50,9 +50,15 @@ sudo crictl pods --name=etcd-member
 
 If no pods are shown, etcd will need to be [investigated](#etcd-is-not-running).
 
-### Unable to SSH into Master Nodes
+### Unable to access the Master Nodes via SSH 
 
-In order to SSH into the master nodes, it is necessary to include an administrator's SSH key during the installation. If SSH authentication is failing, ensure that the proper SSH key is being used.
+In order to SSH into the master nodes as user `core`, it is necessary to include an administrator's ssh-key during the installation. When asked by the installation wizard for the ssh-key, make sure you select your ssh-key from the wizard prompt `~/.ssh/*.pub`. This key will be added to the `core` user's `authorized_keys` file. The public key is placed in authorized_keys by Ignition and is not configured via platform-specific approaches like [AWS key pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html). To verify if you added the correct ssh-key during installation you can use thefollowing command:
```

Comment:
"thefollowing" -> "the following".  Or just end with:

> You can verify the configured SSH key with:
>
> ```sh
> oc get configmap ...
> ```

There's also an `authorized_keys` that still needs backticks.

I'd prefer to use reference-style links for external links (like [we do now for `kubernetes-debug`][1]), but if you're tired of tweaking, I'm ok with the inline link.

[1]: https://github.com/openshift/installer/blame/f0066c4ae126346d986dfbb26f7282bea2819f82/docs/user/troubleshooting.md#L254